files/podman.conf

```diff
@@ -0,0 +1,2 @@
+# https://github.com/containers/libpod/blob/master/troubleshooting.md#5-rootless-containers-cannot-ping-hosts
+net.ipv4.ping_group_range=0 2000000
```

Comment:
I don't think we need to ping hosts from containers in our use cases, so this is unnecessary.